wallet/wallet-cli-lib/src/commands/mod.rs

```diff
@@ -52,6 +51,21 @@ pub enum WalletCommand {
     /// Close wallet file
     CloseWallet,
 
+    /// Encrypts the private keys with a new password, expects the wallet to be unlocked
+    EncryptPrivateKeys {
+        // Optional new password, if not specified will remove any existing encryption.
+        password: Option<String>,
+    },
```

Comment:
I'm a little concerned about the usage here. While the API makes sense to have an `Option<String>` due to the 1:1 mapping, this doesn't make sense in a command line prompt interface. Basically, I'd expect if someone entrs "encrypt_private_keys" in a CLI with no arguemnts, to have an error "please enter the password". 

So while the API is OK, I'd rather have another CLI function `RemovePrivateKeysEncryption`, which does the `None` part, while `EncryptPrivateKeys` enforces providing a password as an argument.

Reply:
added a new `RemovePrivateKeysEncryption` command to the wallet CLI, and made password requred for the `EncryptPrivateKeys` command